doc/modules/compose.rst

```diff
@@ -570,3 +570,31 @@ As an alternative, the HTML can be written to a file using
 
  * :ref:`sphx_glr_auto_examples_compose_plot_column_transformer.py`
  * :ref:`sphx_glr_auto_examples_compose_plot_column_transformer_mixed_types.py`
+
+Visualizing a Chain of Estimators
```

Comment:
I am not sure that change is indeed linked with this PR.
Usually, only modify the docstring in the class. Is not it part of another of your PR @reshamas?

Reply:
Do you mean this PR #18758 ?